README.md

```diff
@@ -1,2 +1,29 @@
 ### Project Title
 A Telegram Bot that reports how many fees are unclaimed from a Uniswap V3 pool
+
+### Description
+The bot is configured at start-up to watch specific pools via environment variables and cli flags (just a list of integers). Only whitelisted accounts are allowed to talk to the bot.
+The bot also sends a message to its users when the unclaimed fees in some pools cross a threshold amount (say 1000$).
+
+### Interaction
+`/status`
+- Prints a list of registered pools and how much $ is claimable
+
+### Future additions
+- In `utils/registered_pools.go`, add parameters for new pools for which the telegram bot is configured to watch
```

Comment:
How do you get these parameters? This should be a config flag (perhaps comma separated to handle the multiple values)

Reply:
It is explained in the next lines of the same README